test/e2e/framework/volume_util.go

```diff
@@ -68,7 +68,7 @@ const (
 	TiB int64 = 1024 * GiB
 
 	// Waiting period for volume server (Ceph, ...) to initialize itself.
-	VolumeServerPodStartupTimeout = 1 * time.Minute
+	VolumeServerPodStartupTimeout = 3 * time.Minute
```

Comment:
is this necessary?

Reply:
In my environment, it takes around 1 minute to set up rbd volume server, as a result, sometime test fails due to the time out. (I hope that my test environment is so much poor.)
Therefore, it would be better to extend it.